test cases/common/47 pkgconfig-gen/meson.build

```diff
@@ -52,3 +52,10 @@ pkgg.generate(
   description : 'A minimalistic pkgconfig file.',
   version : libver,
 )
+
+pkgg.generate(
+  name : 'libhello_nolib',
+  description : 'A minimalistic pkgconfig file.',
+  version : libver,
+  dataonly: true
```

Comment:
You should verify the content of the .pc file is correct too. There are unit tests in run_unittests.py doing that kind of thing.